app/controllers/api/v1/rides_controller.rb

```diff
@@ -1,15 +1,39 @@
 class Api::V1::RidesController < ApiController
-
+  before_action :authorize_admin, only: [:create]
+  
   def index
     render json: Ride.all
   end
 
+  def create
+    ride = Ride.new(ride_params)
+    if ride.save
+      render json: ride 
+    else
+      render json: {errors: ride.errors.full_messages.to_sentence}, status: 401
```

Comment:
we shouldn't need to manually include a status code - here a 401 is technically not the correct status code we'd want. 401 means "Not found", but in a create action we want 422 which means "Unprocessable entity" or something wasn't created correctly